src/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -64,15 +71,17 @@ export class EfisSymbols {
             lat: SimVar.GetSimVarValue('PLANE LATITUDE', 'degree latitude'),
             long: SimVar.GetSimVarValue('PLANE LONGITUDE', 'degree longitude'),
         };
-        const trueHeading = SimVar.GetSimVarValue('PLANE HEADING DEGREES TRUE', 'degrees');
+        const groundSpeed = SimVar.GetSimVarValue('GPS GROUND SPEED', 'knots');
+        // TODO in true mode, this should be true track/heading... should get the values from FM position when implemented
+        const upBearing = (this.trackUpMode && groundSpeed >= 40) ? SimVar.GetSimVarValue('GPS GROUND MAGNETIC TRACK', 'degrees') : SimVar.GetSimVarValue('PLANE HEADING DEGREES MAGNETIC', 'degrees');
```

Comment:
So I think the groundSpeed check is because of #5804? If so, that number is wrong and should be >= 50 knows.

However, ideally this should simply use the ADIRS value instead, because that already handles this ([see PFD code here on how to access the ARINC429 data](https://github.com/davidwalschots/a32nx/blob/79b77e98e6d60b1f5c40e13e84605770a3b30df7/src/instruments/src/PFD/index.jsx#L109)).